content/home/03_expand.md

```diff
@@ -34,23 +34,25 @@ The AusTraits project received **[investment](https://doi.org/10.47486/DP720)**
 We also accept any legacy datasets, either from your archives or transcribed from published data sources.
 
 ### Formalising vocabulary
 The AusTraits team held a series of workshops from 2021-2023 to review and more fully document trait definitions and allowable trait values (*terms* for categorical variables and *ranges* for continuous variables) used to build AusTraits. Our three **[workshops](post/workshop1/)** have, respectively, reviewed 29 trait definitions associated with seeds and germination, 49 trait definitions associated with plant growth form and leaf morphology, and 35 traits associated with fire response and plant regeneration. An additional 114 traits have been reviewed by 1-2 subject experts. These are predominately traits related to plant physiology or floral structure.
 
 ### Establishing the AusTraits Plant Dictionary, APD
-The trait definitions used to map data within AusTraits have been developed into a formal vocabular, the [APD](https://w3id.org/APD). The APD extends the core information required by the AusTraits workflow and releases all trait definitions in both machine-readable RDF representations and human-friendly outputs (html, PDF, and word docuemnt). The pipeline that builds AusTraits requires each trait to have defined units (for numeric traits) and either explicitly allowed (and defined) values (for categorical traits) or an allowed range (for numeric traits). The APD also includes references for traits, links to other trait databases and thesaurus providing data or definitions for the same trait concept, keywords, and hierarchical categories for easy data searching. The APD will shortly be discoverable through the ARDC's Research Vocabulary Australia [portal](https://vocabs.ardc.edu.au/).
+The trait definitions used to map data within AusTraits have been developed into a formal vocabulary, the [APD](https://w3id.org/APD). The APD extends the core information required by the AusTraits workflow and releases all trait definitions in both machine-readable RDF representations and human-friendly outputs (html & csv documents). The dictionary includes trait description metadata required by the pipeline that builds AusTraits including accepted units (for numeric traits) and either explicitly allowed (and defined) values (for categorical traits) or an allowed range (for numeric traits). The APD also includes references for traits, links to other trait databases, ontologies and thesauri providing data or definitions for the same trait concept, keywords, and hierarchical categories for easy data searching. The APD can also be accessed through the ARDC's Research Vocabulary Australia [portal](https://vocabs.ardc.edu.au/viewById/649). For a description of the APD, [see](dio.org/10.1101/2023.06.16.545047)
```

Comment:
mistake in link, should read
For a description of the APD, [see doi: 10.1101/2023.06.16.545047](doi.org/10.1101/2023.06.16.545047).